update_ensembl_data.py

```diff
@@ -40,8 +40,9 @@ def write_genome_data(self, ensembl_id):
         if len(id) != 6:
             raise ValueError(f"Cannot extract six character id from {ensembl_id}")
         if not os.path.exists(dir):
-            raise ValueError(f"Directory {id} corresponding to {ensembl_id} does" +
-            "not exist")
+            raise ValueError(
```

Comment:
We can now remove the `` subprocess.check_call(["black", "-q", path])`` above according to the TODO. Might as well tack a commit on here to do that as well.

Reply:
Ah, yep, I've chucked that in too.